task/09-functions-n-closures-tasks.js

```diff
@@ -104,7 +123,21 @@ function memoize(func) {
  * retryer() => 2
  */
 function retry(func, attempts) {
-    throw new Error('Not implemented');
+    return function() {
+        for(let i = 1; i <= attempts; i++) {
+            try 
+            {
```

Comment:
Curly bracket should be opened on the same line as function or expression declaration

Reply:
Fixed